docs/development.md

```diff
@@ -34,6 +34,34 @@ source ./scripts/env
 
 Make sure to source the script from the repository root.
 
+## Run and connect multiple instances
+
+We provide the `scripts/devnet.ts` tool for running, connecting and
+managing multiple Upstream instances on a local machine.
+
+The devnet tool uses numbers from 1 to 100 to identify and reference instances.
+
+```bash
+scripts/devnet.ts upstream 1
+```
+
+This will an Upstream instance. The instance will be fully initialized with user
```

Comment:
```suggestion
This will start an Upstream instance. The instance will be fully initialized with user
```

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/2726/commits/69c62f827121316755fbe8e5046899ac93498f94